stable/wordpress/README.md

```diff
@@ -55,6 +55,10 @@ The following tables lists the configurable parameters of the WordPress chart an
 | `wordpressFirstName`                 | First name                                 | `FirstName`                                                |
 | `wordpressLastName`                  | Last name                                  | `LastName`                                                 |
 | `wordpressBlogName`                  | Blog name                                  | `User's Blog!`                                             |
+| `databaseName`                       | Database name to create                    | `bitnami_wordpress`                                        |
```

Comment:
The mariadb chart allows you to create a database and user credentials to access the database. Rather than configuring `databaseName`, `databaseUser` and `databasePassword` on the wordpres chart it think `mariadbDatabase`, `mariadbUser` and `mariadbPassword` should be configured on the mariadb chart, similar to how `mariadbRootPassword` is set.